### PR TITLE
Update docker images. Bump some versions. Auto-formatting.

### PR DESCRIPTION
**Pull request by danielhoherd**
The main purpose of this PR is to bump the docker images in values.yaml to use the latest images that have the latest security fixes. See https://github.com/astronomer/issues/issues/2702

- Bumped version of astronomer images in values.yaml
- Bumped CI kind versions to newer patch releases
- Bumped CircleCI machine image of ubuntu, and its version of python
- Bumped CI version of helm
- Bumped to latest ci-helm-release image
- `pre-commit autoupdate` and improve some configs, which is in line with what is being used elsewhere
- Auto-applied `prettier` formatting to some files via vscode plugin

I have deployed this version of the chart to dev as `airflowChartVersion: 0.18.7-build9321` <https://github.com/astronomer/google-environments/commit/f031e1af3ae25b2bb96211379add8450a83099ec>

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -94,15 +94,15 @@ images:
...
     pullPolicy: IfNotPresent
   pgbouncer:
     repository: quay.io/astronomer/ap-pgbouncer
-    tag: 0.11.0
+    tag: 1.8.1
     pullPolicy: IfNotPresent
   pgbouncerExporter:
     repository: quay.io/astronomer/ap-pgbouncer-exporter
-    tag: 0.11.0
+    tag: 0.9.2
```
Comment (kaxil) [inline values.yaml:105]: Are the changes between these versions non-breaking?

**Review by kaxil — APPROVED** [review]

File: values.yaml (hunk 1 above)
Comment (danielhoherd) [reply values.yaml:105]: I'm honestly not sure. I'll deploy this version of the chart to dev so we can see.

File: .circleci/config.yml [hunk 2]
```diff
@@ -95,7 +96,7 @@ jobs:
       - run:
           name: Install Airflow chart
           command: |
-            pyenv global 3.8.5
+            pyenv global 3.9.1
```
Comment (vishwas-astro) [inline .circleci/config.yml:99]: @kaxil @danielhoherd do you know if we are ok with python 3.9 instead of 3.8?

Comment (danielhoherd) [reply .circleci/config.yml:99]: This is just for CI.

Comment (vishwas-astro) [reply .circleci/config.yml:99]: oh got it!

**Comment by vishwas-astro** [comment]
my 👍 if the tests on dev with this new airflow chart build work.

**Review by vishwas-astro — APPROVED** [review]

**Comment by danielhoherd** [comment]
waiting for @ernest-kr to give this the green light

**Review by ernest-kr — APPROVED** [review]

**Pull request merged**